src/components/Onboarding/steps/Finish.js

```diff
@@ -122,18 +124,19 @@ type SocMed = {
   onClick: string => void,
 }
 
+const StyledBox = styled(Box)`
+  cursor: default;
```

Comment:
already default

Reply:
nope, since it's inherited, we have a cursor: text here